src/ui/views/SettingsView/index.tsx

```diff
@@ -1,20 +1,43 @@
-import { Card, CardContent, Divider, Typography } from '@mui/material';
+import { Card, CardContent, Divider, Typography, Button } from '@mui/material';
 import React, { FunctionComponent } from 'react';
 import SettingsIcon from '@mui/icons-material/Settings';
+import { useTranslation } from 'react-i18next';
 import CardTitle from '../../components/CardTitle';
 import MainLayout from '../../layouts/MainLayout';
 
+const locales = {
+  "en": "English" ,
+  "zh-CN": "简体中文",
+};
+
 const SettingsView: FunctionComponent = () => {
+  const { t, i18n } = useTranslation();
+  
   return (
     <MainLayout>
       <Card>
-        <CardTitle icon={<SettingsIcon />} title="Settings" />
+        <CardTitle 
+          icon={<SettingsIcon />} 
+          title={t('SettingsView.Settings')} 
+        />
         <Divider />
         <CardContent>
-          <Typography variant="h6">Todo:</Typography>
+          <Typography variant="h6">{t('SettingsView.LanguageSelector')}</Typography>
+          <ul>
```

Comment:
What do you think about changing the list of buttons to our [<Omnibox/>](https://github.com/ExpressLRS/ExpressLRS-Configurator/tree/master/src/ui/components/Omnibox) component? Currently it looks fine, but we have only 2 languages at the moment. If we get 10 of them or more the current implementation will be not as usable.